charts/exporter/Chart.yaml

```diff
@@ -1,2 +1,3 @@
 name: exporter
-version: v0.0.1
+version: v0.0.2
```

Comment:
I would update the version to `v.1.1.0` to match the current repo version.

Reply:
updated